Brendan initiatives

trippersham opened the pull request.

indiemini committed:
Begin work on Initiatives and InitiativeDetails, using dummy information

indiemini committed:
Add Helvetica LT Std font and fix header-3 padding

indiemini committed:
Fix initiatives card header spacing and begin work on initiative roles

indiemini committed:
Show role name and description in initiative detail cards

indiemini committed:
Initiative role card now links to signup_url

indiemini committed:
Initiative event cards now link to signup_url

MiniguyBrendan commented on client/.eslintrc.json:
```diff
@@ -18,7 +18,7 @@
     "no-console": "off",
     "indent": ["error", 2],
-    "quotes": ["error", "single", { "allowTemplateLiterals": true, "avoidEscape": true }],
+    // "quotes": ["error", "single", { "allowTemplateLiterals": true, "avoidEscape": true }],
```
It's probably odd for me to review my own changes, but, here we are! I disabled this eslint check because the placeholder JSON data I was using was formatted with double quotes, and had single quotes in text content. Do we feel strongly about enforcing single-quote usage?

MiniguyBrendan commented on client/src/components/InitiativeDetail.js:
```diff
@@ -0,0 +1,163 @@
+  
+  if (!detail.loaded)
+  {
+    fetch('http://localhost:8000/initiatives/' + 'id')
```
This should obviously be replaced by the API URL once CORS stuff is sorted out.

MiniguyBrendan commented on client/src/components/Initiatives.js:
```diff
@@ -0,0 +1,61 @@
+function Initiatives() {
+  // const [initiatives, set] = useState(JSON.parse('{}'));
+  document.title = 'HF Volunteer Portal - Initiatives';
+  // fetch('http://localhost:8000/initiatives')
```
Using dummy data, but again, replace this with API URL once CORS stuff is sorted out.

indiemini committed:
Merge branch 'staging' into brendan-initiatives

indiemini committed:
Merge branch 'staging' into brendan-initiatives

indiemini committed:
Use relative routing for API, use real data instead of dummy data, use loading spinner properly

indiemini committed:
Remove excess logging

indiemini committed:
Swap out the dropdown Initiatives navbar menu for a basic link

we'll bring the drop-down back when we have a nicer way to pre-fetch everything required from the API.
